lib/si-filesystem/src/client.rs

```diff
@@ -239,26 +246,27 @@ impl SiFsClient {
         {
             return Ok(cached_value);
         }
+        log::trace!("making request to {url}");
 
+        let start = Instant::now();
         let mut request_builder = self.client.get(url.clone()).bearer_auth(&self.token);
         request_builder = if let Some(query) = query.clone() {
             request_builder.query(&query)
         } else {
             request_builder
         };
 
-        let start = Instant::now();
-        let response = request_builder.send().await?;
+        let response = tokio::spawn(async move { request_builder.send().await }).await??;
```

Comment:
I'm not totally sure I understand the reason for spawning this task and then awaiting it. Could this have been from some other work that didn't get reverted--or maybe I don't see it here :)

Reply:
oh! good call. I was experimenting with reqwest which is just *slow* for some reason. Requests that you can make in 50ms with `curl` take 250ms with reqwest and I can't figure out why. I didn't mean to leave this in. I'll clean it up in a separate PR